packages/oauth/src/providers/strava.ts

```diff
@@ -58,7 +58,7 @@ export class StravaAuth<_Auth extends Auth = Auth> extends OAuth2ProviderAuth<
 
 	private validateAuthorizationCode = async (
 		code: string
-	): Promise<[StravaUser, StravaTokens]> => {
+	): Promise<[stravaUser: StravaUser, stravaTokens: StravaTokens]> => {
```

Comment:
Out of interest, because I don't know the answer: why do you name the params in the generic? (I am still somewhat rusty at Typescript).

Reply:
Mostly for consistency - other APIs (like `createOAuth2AuthorizationUrl()`) uses `string` so it makes sense to name them